src/FINDOLOGIC/Export/Data/Image.php

```diff
@@ -57,6 +57,15 @@ public function getUrl()
         return $this->url;
     }
 
+    private function setUrl($url)
+    {
+        $url = DataHelper::checkForEmptyValue($url);
+
+        $url = DataHelper::validateUrl($url);
```

Comment:
URL validation is still here, impacting CSV as well.

Reply:
There also should be a test case in `CSVSerializationTest` to check that - having one like this would have prevented leaving validation in here by accident.

Reply:
Done